### DISCUSSION TITLE
Feature Request: Entry view enhancement

### DISCUSSION THREAD

**Discussion by jbrew30** [trendyminds/isolate#31] [Ideas]
Apply the same filtering, search, layout, and customization abilities that the current craft entries page has to the isolated entry view. 

![Screen-Shot-2020-06-16-at-2 53 34-PM](https://user-images.githubusercontent.com/22452733/84815838-a04b1900-afe1-11ea-83af-2808c54d4c5c.jpg)

![Screen-Shot-2020-06-16-at-3 00 44-PM](https://user-images.githubusercontent.com/22452733/84817036-79d9ad80-afe2-11ea-8290-c04c4a9c5d2b.jpg)

**Comment by aaronbushnell** [comment]
Hello! We've moved all open Issues to **Discussions**.

Unfortunately, due to client workload we often find ourselves unable to implement new features or resolve issues in a timely manner. Switching to Discussions allows better categorization to separate "Feature Requests" from bugs. It also allows the community to better suggest/provide solutions when we're unable to do so.

Hope Isolate continues to be useful for you!
